apps/dot-voting/app/components/VotingOptions.js

```diff
@@ -52,16 +57,21 @@ class VotingOptions extends React.Component {
             to={{ value: safeDiv(parseInt(option.value, 10), totalSupport) }}
             native
           >
-            {({ value }) => (
-              <VotingOption
-                fontSize={this.props.fontSize}
-                valueSpring={value}
-                percentage={safeDiv(parseInt(option.value, 10), totalSupport)*100}
-                color={color}
-                userVote={(voteWeights.length && displayYouBadge) ? Math.round(voteWeights[i]) : -1}
-                {...option}
-              />
-            )}
+            {({ value }) => {
+              const percentage = safeDiv(parseInt(option.value, 10), totalSupport)
+              const allocation = symbol ? `${BigNumber(balance).times(percentage).dp(3).toString()} ${symbol}` : ''
```

Comment:
I think these two calculations will eventually end up in the app state reducer, which I believe is the best place for them. There, they can be included in `vote.options` reducing the complexity here. But that's for another PR.

Here, you can check `showAllocations` instead of `symbol`, and if it's true, pass it as a string with the parentheses already in place; if it's false, pass an empty string. That way you remove the check you have to make in `VotingOption`.